app/src/main/java/de/westnordost/streetcomplete/util/location/FineLocationManager.kt

```diff
@@ -98,9 +98,13 @@ private fun Location.isBetterThan(previous: Location?): Boolean {
     if (previous == null) return true
 
     // Check whether the new location fix is newer or older
-    val timeDelta = this.time - previous.time
-    val isMuchNewer = timeDelta > TWO_MINUTES
-    val isMuchOlder = timeDelta < -TWO_MINUTES
+    // we use elapsedRealtimeNanos instead of epoch time because some devices have issues
+    // that may lead to incorrect GPS location.time (e.g. GPS week rollover, but also others)
+    // the use of nanoseconds is necessary because it iss the only way to get
```

Comment:
Typo: iss → is

Reply:
https://github.com/streetcomplete/StreetComplete/commit/ac56d0866a9c968e41f4b406d28f13be5e8cf155